include/openrave/units.h

```diff
@@ -37,11 +37,13 @@ enum LengthUnit : int8_t
     LU_Nanometer = -9,
     LU_Inch = 0x10,
     LU_Foot = 0x11,
+    LU_TenthMillimeter = 0x12,
```

Comment:
i would rather call it "100Micrometer", and the value should be -4

Reply:
why do we need this by the way?

Reply:
We have a python version unit conversion function (`GetUnitConversionScale`), which supports `0.1mm`. Do we want to support it as well here?